tamzi/FlixCompanion
# Dev

---
name: Pull request
about: Update dependencies
label: 'triage me'
Fixes [Referenced Issue #](https://github.com/droidconKE/chai/issues)_.


---
## How can this PR be tested?
>check out and rumn

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:


Is this your first Pull Request?

Fixes #<issue_number_goes_here> 🦕

# Screenshot
> For PRs that change the UI, attach a before/after screenshot.
> **[REMOVE THIS SECTION IF IT DOES NOT APPLY TO THIS PR]**

![]()
